README.md

```diff
@@ -1,143 +1,198 @@
-
-No warranty is provided or implied.
-
-## Installation
-
-<!-- ### Requirements
-
-The following are required to build Juvix:
-
-- [libff](https://github.com/scipr-lab/libff)
-- [libsecp256k1](https://github.com/bitcoin-core/secp256k1)
-- [Openssl Libssl API](https://wiki.openssl.org/index.php/Libssl_API) -->
-
-
-[Stack](https://haskellstack.org) required. 8GB RAM required for `stack` installation.
+[![stability-experimental](https://img.shields.io/badge/stability-experimental-orange.svg)](https://github.com/mkenney/software-guides/blob/master/STABILITY-BADGES.md#experimental)
+![GitHub release (latest SemVer including pre-releases)](https://img.shields.io/github/v/release/anoma/juvix?include_prereleases)
+====
+
+Description
+-----------
+
+<img align="right" width="300" height="300" src="Juvix_logo.png">
+
+Juvix is a dependently functional programming language for writing
+efficient formally-verified smart contracts, which can be
+deployed to various distributed ledgers. Juvix addresses many issues that we have
+experienced while trying to write and deploy decentralised
+applications present in the ecosystem of smart-contracts: 
+- the difficulty of
+adequate program verification, 
+- the ceiling of compositional
+complexity, 
+- the illegibility of execution costs, and
+- the lock-in to
+particular backends.
+
+The Juvix compiler synthesises a high-level frontend syntax with
+support for dependent-linearly types and several other cutting-edge
+research ideas from programming language design and type theory. For
+more details, see [the Juvix language
+reference WIP](./doc/reference/language-reference.pdf).
+
+Prerequisites
+--------------
+
+* Install [Stack](https://haskellstack.org):
 
   - For Ubuntu        : `apt install stack`
   - For Debian        : `apt install haskell-stack`
   - For Arch Linux    : `pacman -S stack`
   - For macOS : `brew install haskell-stack`
-  - For Windows, following the instructions [here](https://docs.haskellstack.org/en/stable/install_and_upgrade/#windows).
-<!-- - **libsecp256k1**
-  - For Ubuntu/Debian : `apt install libsecp256k1-dev`
-  - For Arch Linux : `pacman -S libsecp256k1`
-  - For macOS : `brew tap cuber/homebrew-libsecp256k1 && brew install libsecp256k1`
-- **Openssl Libssl API**
-  - For Ubuntu/Debian : `apt install libssl-dev`
-  - For Arch Linux : `pacman -S openssl`
-  - For macOS : `brew install openssl` -->
+  - For Windows, following the instructions
+  [here](https://docs.haskellstack.org/en/stable/install_and_upgrade/#windows).
+  
+  It is required at least 8GB RAM for `stack` installation.
+
+* You might need to install the following third-party libraries.
+
+  - [libff](https://github.com/scipr-lab/libff),
+  - [libsecp256k1](https://github.com/bitcoin-core/secp256k1), and
+  - [openssl libssl](https://wiki.openssl.org/index.php/Libssl_API).
+
+
+Installation
+------------
+
+1. To install Juvix, you can download its sources using
+   [Git](http://git-scm.com/) from the [Github
```

Comment:
I don't think we need to mention git cuz if the person has never used git, they will have a hard contributing to Juvix anyway.

Reply:
The readme is not just for developers, for contributors, we have this other page, CONTR..